content/800-guides/120-using-prisma-in-cursor.mdx

```diff
@@ -0,0 +1,512 @@
+---
+title: "Using Prisma ORM in Cursor"
+metaTitle: "Using Prisma ORM in Cursor"
+description: "Get the most out of Prisma ORM when using Cursor AI editor"
+sidebar_label: "Using Prisma ORM in Cursor"
+image: "/img/guides/using-prisma-orm-in-cursor.png"
+tags:
+- Cursor
+- AI Editor
+- AI
+---
+
+[Cursor](https://www.cursor.com/) is an AI-powered code editor designed to boost productivity by automating repetitive coding tasks. When paired with Prisma ORM, a robust and type-safe toolkit for database workflows, it becomes a powerful solution for managing and optimizing database schemas, queries, and data seeding.
+
+This guide provides detailed instructions for effectively using Prisma with Cursor to:
```

Comment:
Perhaps somewhere here in the intro (perhaps at the end of the intro) we can mention that a bunch of these ideas may apply to other AI editors as well, and that people should tweet at us (and equivalent on other platforms) to tell us what other editors and toold people want us to create content on?